pkg/controller/garbagecollector/garbagecollector.go

```diff
@@ -515,45 +556,63 @@ func (gc *GarbageCollector) attemptToDeleteItem(ctx context.Context, item *node)
 		// the GraphBuilder can add "virtual" node for an owner that doesn't
 		// exist yet, so we need to enqueue a virtual Delete event to remove
 		// the virtual node from GraphBuilder.uidToNode.
-		klog.V(5).Infof("item %v not found, generating a virtual delete event", item.identity)
+		logger.V(5).Info("item not found, generating a virtual delete event",
+			"item", item.identity,
```

Comment:
"item"?

Reply:
Same as above - switching to item would be a change